robottelo/utils/url.py

```diff
@@ -0,0 +1,9 @@
+from urllib.parse import urlparse
```

Comment:
An alternative to creating a new file for a single helper function is to just put it in the __init__.py for this directory.
non-blocking